README.md

```diff
@@ -16,9 +16,132 @@ Or install it yourself as:
 
     $ gem install mergeq
 
-## Usage
+Setup your mergeq directory
 
-TODO: Write usage instructions here
+    $ cd your_project
+    $ mkdir .mergeq
+
+Ignore the merging lock file in git
+
+    $ echo ".mergeq/merging" >> .gitignore
```

Comment:
:information_desk_person: It would be nice to just factor this all out into an idempotent setup script